01-intro.Rmd

```diff
@@ -3,16 +3,44 @@
 ottrpal::set_knitr_image_path()
 ```
 
-# Introduction
+# (PART\*) Introduction {-}
 
+# Why this new policy? {#why-this-new-policy}
 
-## Motivation
+```{r, echo=FALSE, fig.alt='We are on the first step of the pathway- understading the motivations of the policy.', out.width = '80%', fig.align = 'center'}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1luFoDzF6aDJEebbL6iWoJ_s8s9nQnaWLL5jghbmWdak/edit#slide=id.g16b04bd80e5_1_103")
+```
+
+First we will discuss the motivations behind the new data management and sharing policy that will go into effect for (most) grant proposals submitted to the NIH after January 25, 2023.
+
+So why is the NIH doing this? There are several reasons why sharing data can be beneficial to the scientific community.
+
+- **Supports transparency** - Sharing data provides more clarity about how studies are performed. 
+-  **Encourages reproducibility and rigor** - Having the data accessible, allows others to try to reproduce study findings. This can further enable studies that may replicate or validate the initial findings with different data. 
```

Comment:
```suggestion
- **Encourages reproducibility and rigor** - Having the data accessible allows others to try to reproduce study findings. This can further enable studies that may replicate or validate the initial findings with different data. 
```

Reply:
Extra space